specs/source.go

```diff
@@ -20,8 +25,9 @@ type Source struct {
 	// For the gRPC registry the path will be the address of the gRPC server: host:port
 	Path string `json:"path,omitempty"`
 	// Registry can be github,local,grpc.
-	Registry    Registry `json:"registry,omitempty"`
-	Concurrency uint64   `json:"concurrency,omitempty"`
+	Registry            Registry `json:"registry,omitempty"`
+	TableConcurrency    uint64   `json:"table_concurrency,omitempty"`
```

Comment:
Just a thought: we could keep this change backwards-compatible by keeping `concurrency` as an option, but using it to set `table_concurrency` if it is provided.

Reply:
let's just break it. I don't think anyone is using this at the moment anyway given we just released it last week.

Reply:
It would open up more confusion, ie. should check and error out if both of those are set by the user.

Reply:
if someone will use `concurrency` it will error out by default because we check if there are any unknown fields via built-in `json` decoder